plugins/inputs/opcua_listener/README.md

```diff
@@ -0,0 +1,241 @@
+# OPC UA Client Listener Input Plugin
+
+The `opcua_listener` plugin subscribes to data from OPC UA Server devices.
+
+Telegraf minimum version: Telegraf 1.24
+Plugin minimum tested version: 1.24
```

Comment:
Technically this won't make it in't 1.24 as that version is already released. New features will go into 1.25.